cpp/ql/lib/semmle/code/cpp/models/implementations/StructuredExceptionHandling.qll

```diff
@@ -1,9 +1,29 @@
 import semmle.code.cpp.models.interfaces.Throwing
 
-class WindowsDriverFunction extends ThrowingFunction {
-  WindowsDriverFunction() {
+/**
+ * The default behavior for SEH is any function may (conditionally) raise an exception.
+ * NOTE: this can be overriden by for any specific function to make in
+ * unconditional or non-throwing. IR generation will enforce
+ * the most strict interpretation.
```

Comment:
Where does this enforcement happen?

Reply:
https://github.com/microsoft/codeql/blob/e86c7644d8a39f95f97d24f93c6db403191a9b99/cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedCall.qll#L89